helpers/external-api/v1.ts

```diff
@@ -751,9 +751,9 @@ export interface PublicApiColumn {
    */
   formula?: string;
   /**
-   * Default value formula for the column.
+   * Suggested value value formula for the column.
    */
-  defaultValue?: string;
+  suggestedValue?: string;
```

Comment:
This is a generated code file copied over from `coda`, and confusingly this field actually refers something different. So can just revert this file.